apps/portals/elportal/src/pages/HomePageV2.tsx

```diff
@@ -1,50 +1,23 @@
-import Ecosystem from '@sage-bionetworks/synapse-portal-framework/components/csbc-home-page/Ecosystem'
-import Header from '@sage-bionetworks/synapse-portal-framework/components/Header'
 import { SectionLayout } from '@sage-bionetworks/synapse-portal-framework/components/SectionLayout'
 import React from 'react'
 import {
-  CardContainerLogic,
-  Goals,
-  Markdown,
-  RssFeedCards,
-  SynapseConstants,
-  UserCardListRotate,
   RecentPublicationsGrid,
   ImageCardGridWithLinks,
 } from 'synapse-react-client'
 import ELContributeYourData from '@sage-bionetworks/synapse-portal-framework/components/elportal/ELContributeYourData'
-import {
-  partnersSql,
-  peopleSql,
-  topPublicationsSql,
-  whatWeDoSql,
-} from '../config/resources'
+import { topPublicationsSql, whatWeDoSql } from '../config/resources'
+import { Box } from '@mui/material'
 
 export default function HomePage() {
   return (
     <>
-      <Header />
-      <SectionLayout
-        title={'About the Portal'}
-        centerTitle
-        ContainerProps={{ className: 'home-spacer' }}
-      >
-        <Markdown
-          ownerId={'syn27229419'}
-          wikiId="626030"
-          loadingSkeletonRowCount={10}
-        />
-      </SectionLayout>
-      <SectionLayout ContainerProps={{ className: 'home-spacer' }}>
-        <ImageCardGridWithLinks
-          sql={whatWeDoSql}
-          title="What We Do"
-          summaryText="We provide multi-omic datasets, software tools, and publications that empower researchers to discover the latest health-promoting therapeutics."
-        />
-      </SectionLayout>
-      <SectionLayout ContainerProps={{ className: 'home-spacer' }}>
-        <Goals entityId={'syn51449135'} />
-      </SectionLayout>
+      {/* <SectionLayout ContainerProps={{ className: 'home-spacer' }}> */}
```

Comment:
If the SectionLayout should be removed, then remove it instead of commenting it.

If it needs to be adjusted, prefer you keep it and make the adjustments later.